### ISSUE TITLE
Allow multiple notification addresses and notify several addresses

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#94]
```
Ability to share usage of your phone with close friends would be neat...
```

Original issue reported on code.google.com by `me.detected` on 10 Feb 2011 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
Sounds nice at first. But I am against this type of feature. Because then these 
other JIDs will have full control and get all incoming SMS if we don't 
implement some kind of ACLs which would bloat the whole code.  
A better way would be if you create a special jabber account that you share 
with your friends which is the notify address. Then you have the same 
functionality that you desire.
```

Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
ACL sounds nice, at least it would be great if there were two different 
'rights' for users: r and r+w
```

Original comment by `me.detected` on 10 Feb 2011 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
P.S. GTalk users cannot log into several accounts simultaneously 
```

Original comment by `me.detected` on 10 Feb 2011 at 6:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 9:42
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Sure they can. I've been able to connect to on my phone's Gtalk client and on 
my computer's desktop client, and on my Gmail's gtalk client and on my Pidgin's 
Gtalk client. 

Did it recently change?
```

Original comment by `i.vita...@gmail.com` on 9 Mar 2011 at 9:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 Mar 2011 at 3:02
- Changed title: **feature request: multiple notification addresses / notify several addresses**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 26 Aug 2011 at 8:05
- Changed title: **Allow multiple notification addresses and notify several addresses**

**Comment by GoogleCodeExporter** [comment]
```
Issue 271 has been merged into this issue.
```

Original comment by `fschm...@gmail.com` on 21 May 2012 at 10:29

**Comment by GoogleCodeExporter** [comment]
```
Will be integrated into the new UI
```

Original comment by `Florent....@gmail.com` on 21 May 2012 at 6:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 10 Jun 2012 at 3:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
